packages/graphql-ui/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "extends": "../../tsconfig.base.json",
```

Comment:
You should consider instead extending https://github.com/tsconfig/bases/blob/main/bases/create-react-app.json

It contains most of the configuration you need, so it can really cut down on boilerplate. You can see how this is done in https://github.com/neo4j/graphql/blob/dev/examples/neo-push/client/src/tsconfig.json - also check the `package.json` file there as you need to install a little dev dependency.